dcpy/lifecycle/distribute/socrata.py

```diff
@@ -78,25 +78,16 @@ def _dist_from_local(
             raise Exception("Only shapefiles have been implemented so far")
 
 
-@app.command("from_s3")
+@socrata_app.command("from_s3")
 def _dist_from_s3(
-    product_name: str = typer.Option(
```

Comment:
Something that seems obvious in retrospect: required args shouldn't be typer.Options. It looks like we `should` be using [annotations](https://typer.tiangolo.com/tutorial/arguments/help/#add-a-help-text-for-a-cli-argument) for these. TODO for later.

Reply:
So their docs have

`name: Annotated[str, typer.Argument(help="The name of the user to greet")]`

but I think you can also just do

`name: str = typer.Argument(help="The name of the user to greet")`